foyer/forcefields/oplsaa.xml

```diff
@@ -148,9 +148,9 @@
   <Type name="opls_148" class="CT" element="C" mass="12.01100"   def="[C;X4]([C;%opls_145])(H)(H)H" desc="toluene CH3" overrides="opls_149,opls_135" doi="10.1021/ja9621760"/>
   <Type name="opls_149" class="CT" element="C" mass="12.01100"   def="[C;X4]([C;%opls_145])(H)(H)*" desc="ethyl benzene CH2" overrides="opls_136" doi="10.1021/ja9621760"/>
   <Type name="opls_150" class="opls_150" element="C" mass="12.011"/>
-  <Type name="opls_151" class="opls_151" element="Cl" mass="35.453"/>
-  <Type name="opls_152" class="opls_152" element="C" mass="12.011"/>
-  <Type name="opls_153" class="opls_153" element="H" mass="1.008"/>
+  <Type name="opls_151" class="opls_151" element="Cl" mass="35.453"    def="[Cl][C;X4]" desc="Cl in alkyl chlorides"  />
+  <Type name="opls_152" class="opls_152" element="C" mass="12.011"   def="[C;X4](Cl)(H)(H)*" desc="RCH2Cl in alkyl chlorides"  />
```

Comment:
I believe this same atomtype can also be used for trichloro methane (see [here](https://github.com/mosdef-hub/foyer/blob/master/foyer/opls_validation/trichloromethane/trichloromethane.top))

I would probably make use of an `or` operator here so something along the lines of:
`[C;X4](Cl)([Cl,H])([Cl,H,C])[Cl,H,C]`

This also applies to the hydrogen definition below. If you go into the opls_validation folder and simply `ls *chlor*` you can get a bunch of examples of correctly atom typed chlorinated systems.